packages/expo-modules-core/android/src/main/cpp/JavaScriptModuleObject.cpp

```diff
@@ -48,6 +49,17 @@ std::shared_ptr<jsi::Object> JavaScriptModuleObject::getJSIObject(jsi::Runtime &
   return jsiObject;
 }
 
+void JavaScriptModuleObject::exportConstants(
+  jni::alias_ref<react::NativeMap::javaobject> constants
+) {
```

Comment:
One line doesn't look so bad here 😉 
```suggestion
void JavaScriptModuleObject::exportConstants(jni::alias_ref<react::NativeMap::javaobject> constants) {
```